examples/medplum-demo-bots/src/health-gorilla/samples.md

```diff
@@ -0,0 +1,241 @@
+# Sample Data
+
+[Health Gorilla](https://www.healthgorilla.com/) is an interoperability platform providing permitted access to actionable patient data. When placing orders and receiving results from Health Gorilla in FHIR, you'll need to ensure the following data is populated.
+
+Apart from standard FHIR fields, there are some specifics with regards to the Health Gorilla integration that are required for an integration to be successful. This guide documents required fields. Health Gorilla publishes a list of requirements on their [FHIR Profile](https://developer.healthgorilla.com/docs/fhir-profiles) page which is more comprehensive, but this material is meant to give you the minimal set of required fields to send orders and receive results.
+
+Coming soon: Medplum FHIR Profile specifically for Health Gorilla
+
+## Patient
+
+The patient must have a `country` in their address and a Medical record number as specified in the example. They also must have an `email` address under `telecom` as shown.
+
+```json
+{
+  "resourceType": "Patient",
+  "identifier": [
+    {
+      "type": {
+        "coding": [
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
+            "code": "MR",
+            "display": "Medical record number"
+          }
+        ],
+        "text": "Medical record number"
+      },
+      "value": "55555558"
+    },
+    {
+      "system": "https://www.healthgorilla.com",
+      "value": "df39fa64131a52bfae9848ae"
+    }
+  ],
+  "name": [
+    {
+      "given": ["Homer"],
+      "family": "Simpson"
+    }
+  ],
+  "gender": "male",
+  "birthDate": "1962-06-15",
+  "address": [
+    {
+      "line": ["2425 Sutter St"],
+      "city": "San Francisco",
+      "state": "CA",
+      "postalCode": "94115",
+      "country": "US"
+    }
+  ],
+  "telecom": [
+    {
+      "system": "email",
+      "use": "home",
+      "value": "homer@example.com"
+    },
+    {
+      "system": "phone",
+      "use": "home",
+      "value": "4158006122"
+    }
+  ]
+}
+```
+
+## Practitioner
+
+The Practitioner must have a valid NPI, represented as an identifier as well as a Health Gorilla identifier.
+
+```json
+{
+  "resourceType": "Practitioner",
+  "name": [
+    {
+      "given": ["Alice"],
+      "family": "Smith"
+    }
+  ],
+  "identifier": [
+    {
+      "value": "80ef2c64524ee10b76af5126",
+      "system": "https://www.healthgorilla.com"
+    },
+    {
+      "system": "http://hl7.org/fhir/sid/us-npi",
```

Comment:
At present, the only value we're using is the value from `"system": "https://www.healthgorilla.com"`, we're not using NPI.

We can look into whether NPI search is an option.  I agree that would be simpler.